common/eq_constants.h

```diff
@@ -1131,4 +1131,12 @@ namespace LeadershipAbilitySlot {
 	constexpr uint16 HealthOfTargetsTarget = 14;
 }
 
+#define PARCEL_SEND_ITEMS    0
+#define PARCEL_SEND_MONEY    1
+#define PARCEL_MONEY_ITEM_ID 99990 // item id of money
+#define PARCEL_MAX_ITEMS     RuleI(Parcel, ParcelMaxItems)
```

Comment:
`PARCEL_MAX_ITEMS` is compile time

Rule value will not change dynamically

Reply:
Updated this define.